account_mail_templates/templates/account_mail_templates.xml

```diff
@@ -7,34 +7,32 @@
     name="Invoice: Send by email"
 >
     <t t-call="base_templates.mail_body_layout_basic">
-        <p name="p_refer">
-            <t t-call="base_templates.msg_partner_refer">
-                <t t-set="partner" t-value="record.partner_id"/>
+        <t t-call="base_templates.msg_partner_refer">
+            <t t-set="partner" t-value="record.partner_id"/>
+        </t>
+        <t t-call="base_templates.msg_doc_description_with_amount">
+            <t t-set="doc_text">
+                <span name="span_doc_text">Here is your invoice</span>
             </t>
-        </p>
-        <p name="p_description">
-            <t t-call="base_templates.msg_doc_description_with_amount">
-                <t t-set="doc_text">
-                    <span name="span_doc_text">Here is your invoice</span>
-                </t>
-                <t t-set="doc_number" t-value="record.number"/>
-                <t t-set="origin" t-value="record.origin"/>
-                <t
-                    t-set="amount"
-                    t-value="format_amount(record.amount_total, record.currency_id)"
-                />
+            <t t-set="doc_number" t-value="record.number"/>
+            <t t-set="origin" t-value="record.origin"/>
+            <t
+                t-set="amount"
+                t-value="format_amount(record.amount_total, record.currency_id)"
+            />
+            <t t-set="extra_description_el">
+                <span
+                    name="span_paid_text"
+                    t-if="record.state == 'paid'"
+                >
+                    This invoice is already paid.
+                </span>
+                <span name="span_not_paid_text" t-else="">
```

Comment:
```suggestion
                <span 
                    name="span_not_paid_text" 
                    t-else=""
                >
```

Reply:
We indentate like that, when line becomes too long. Does not mean, always.

Reply:
I know :) condition above seemed similar, so suggested to use same formatting.